community/contribute/adding-docs-guide.md

```diff
@@ -23,34 +23,37 @@ Your repository containing the documentation has to...
 
 The documentation files have to be in markdown format and...
 
-- comply with [SCS licensing guidelines](../github/dco-and-licenses.md)
+- comply [SCS licensing guidelines](https://github.com/SovereignCloudStack/docs/blob/main/community/github/dco-and-licenses.md)
 - match our
-  - [markdown file structure guideline](../contribute/doc-files-structure-guide-md)
+  - [markdown file structure guideline](https://github.com/SovereignCloudStack/docs/blob/main/community/contribute/doc-files-structure-guide-md)
   - linting Rules
-  - [styleguide](../contribute/styleguide.md)
+  - [styleguide](https://github.com/SovereignCloudStack/docs/blob/main/community/contribute/styleguide.md)
 
 ### Step 2 – Adding your repo to the docs.json
 
 File a Pull Request within the [docs-page](https://github.com/SovereignCloudStack/docs-page) repository and add your repo to the docs.package.json:
 
 ```json5
```

Comment:
As you dropped the comments from here to provide valid (plain) JSON, I think my change regarding JSON5 can be reverted:
```suggestion
```json
```